app/models/infra_conversion_job.rb

```diff
@@ -450,13 +460,30 @@ def poll_transform_vm_complete
       raise migration_task.options[:virtv2v_message]
     when 'succeeded'
       update_migration_task_progress(:on_exit)
-      queue_signal(:poll_inventory_refresh_complete)
+      queue_signal(:inventory_refresh)
     end
   rescue StandardError => error
     update_migration_task_progress(:on_error)
     abort_conversion(error.message, 'error')
   end
 
+  def inventory_refresh
+    update_migration_task_progress(:on_entry)
+    if migration_task.options[:destination_vm_uuid].present?
+      target = InventoryRefresh::Target.new(
+        :association => :vms,
+        :manager_ref => {:ems_ref => destination_vm_ems_ref(migration_task.options[:destination_vm_uuid])},
+        :manager     => migration_task.destination_ems
+      )
+      EmsRefresh.queue_refresh_task(target)
```

Comment:
If you want to use the task to track the state of the refresh you can store the task_id in the context and use that for the "wait for refresh to complete" step.

If you just want to wait to see the VM in the database then you don't need to create a task and can just use `EmsRefresh.queue_refresh(target)` which will save a task record.